modules/microshift-preparing-to-make-app-rpms.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-preparing-to-make-app-rpms_{context}"]
+= Preparing to make application RPMs
+
+To build your own RPMs, choose a tool of your choice, such as the the `rpmbuild` tool, and initialize the RPM build tree in your home directory. The following is an example procedure. As long as your RPMs are accessible to Image Builder, you can use the method you prefer to build the application RPMs.
```

Comment:
I would remove this section completely. I do not think we need to show users how to build RPMs.

Reply:
No, lets keep it here, to make the tutotrial self contained and working.